tests/vm_test.go

```diff
@@ -978,6 +979,49 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			})
 		})
 	})
+
+	Context("with oc/kubectl", func() {
+		var vm *v1.VirtualMachine
+		var err error
+		var vmJson string
+
+		// Get proper commandline client: oc or kubectl
+		k8sClient := tests.GetK8sCmdClient()
+
+		AfterEach(func() {
+			if vmJson != "" {
+				err = os.Remove(vmJson)
+				Expect(err).ToNot(HaveOccurred())
+				vmJson = ""
+			}
+		})
+
+		It("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]should create VM only once", func() {
```

Comment:
rfe_id should be used in "Describe" section (it can combine all tests that have one rfe_id)

 In "It" we need to add test_id:243. Also in negative scenarios we need to add: [posneg:negative], so this string should be:
		It("[test_id:243][posneg:negative]should create VM only once", func() {
￼

Reply:
Thanks for copy paste solution.